Simplified/NYPLRemoteViewController.m

```diff
@@ -149,19 +147,25 @@ - (void)connectionDidFinishLoading:(__attribute__((unused)) NSURLConnection *)co
 {
   [self.activityIndicatorView stopAnimating];
   self.activityIndicatorLabel.hidden = YES;
-  
-  if ([(NSHTTPURLResponse *)self.response statusCode] != 200
-      && ([self.response.MIMEType isEqualToString:@"application/problem+json"]
-          || [self.response.MIMEType isEqualToString:@"application/api-problem+json"])) {
-    NYPLProblemDocument *problem = [NYPLProblemDocument problemDocumentWithData:self.data];
-    NYPLAlertController *alert = [NYPLAlertController alertWithTitle:problem.title message:problem.detail];
-    [alert setProblemDocument:problem displayDocumentMessage:NO];
+  BOOL mimeTypeMatches = [self.response.MIMEType isEqualToString:@"application/problem+json"] ||
+    [self.response.MIMEType isEqualToString:@"application/api-problem+json"];
```

Comment:
I don't understand the details of this PR, but I think this method is the crucial part of the change, so I'm focused on it and I don't see the change. Previously you were using `problem.detail` -- the piece of text I'd expect to see but wasn't seeing -- and now you're using `pDoc.detail` in the same place. Is there something different about the _surrounding_ code that makes the detail text get rendered?

If the software crashes, I don't think you'll get a problem document -- just the string "Internal Server Error". It would be nice to notice that, but I realize that's not a lot to work with. We have a ticket to improve our handling of crashes (https://jira.nypl.org/browse/SIMPLY-1840), and maybe we could send you a problem detail document even when there's an ISE.

Reply:
The primary change lies in `setProblemDocument` whereby the old version omitted the details and the updated one includes it. That particular place you mentioned will not see any change, as you noted.

Reply:
Great, got it, thanks.